### ISSUE TITLE
Button images on Android not working

### ISSUE THREAD

**Issue by fidoboy** [AppGyver/input-plugin#3]
I'm trying to add custom images on Android, using SVG images and native-style android.css but i can't get it working (it works fine on iOS). I don't know if I'm doing something wrong or if there is a bug. Could you please test it and/or provide one working example on Android platform?

Thanks in advance,

**Comment by satusuuronen** [comment]
We'll be doing a development sprint where we use the native input plugin heavily ourselves in the near future. After that, we'll take a look at these and other issues that might've come up during our development. So it'll be a few weeks, but in the meantime, be encouraged to develop your own fork and submit PRs.

**Comment by fidoboy** [comment]
Thanks for the update @satusuuronen I've already submitted one PR. But unfortunately there are serious issues with iOS that involves the native Appgyver webViews and I don't know how to solve it. It could be amazing if this plugin is revised soon... 

About the SVG issues with Android I'm afraid that it's beyond of my knowledge and it requires some Pixate engine fine tuning

**Comment by harsaharsa** [comment]
Yup, we'll go through these ASAP as we get the immediate backlog cleared. As always, the more detailed the repro case (with actual code), the faster for us it is to fix these issues.

**Comment by fidoboy** [comment]
Partially fixed in my own fork. I've replaced the button with ImageButton that allows to use the image selector from CSS native styles. Now SVG images are being displayed fine. Currently I'm trying to fix a new problem with background color... I'll report back here my progress and I'll create a PR when it's ready

**Comment by fidoboy** [comment]
Background color and other issues related to buttons are fixed now in my fork. However it's still a work in progress because I'm trying to enhance the plugin by solving many problems and also adding new functions and events. Any help could be more than welcome now, my knowledge with native code is limited and there many issues related to Steroids native part that I can't figure out how to resolve... Let's see if there is anyone into the Appgyver community that wants to join here and to help with this plugin; I believe that it could be helpful for everyone.

**Comment by fidoboy** [comment]
Latest changes from @pentateu did not solve the Android issue with image buttons. It's fixed in my fork
